tests-arquillian/src/test/java/org/jboss/weld/tests/interceptors/producer/InterceptionFactoryTest.java

```diff
@@ -65,6 +66,13 @@ public void testFooClassLevelBinding(@Produced("classLevel") Foo foo) {
     public void testFooEjbInterceptors(@Produced("ejbInterceptors") Foo foo) {
         assertEquals("Hello pong", foo.ping());
     }
+    
+    @Test
+    public void testFooNoInterceptors(@Produced("empty") Foo foo) {
+        // Note that the produced Foo is @Dependent
+        assertEquals(Foo.class, foo.getClass());;
+        assertEquals("pong", foo.ping());
```

Comment:
`assertEquals(Foo.class, foo.getClass())`. The produced `Foo` is `@Dependent` and so there is no client proxy and we can be sure the instance is the result of the producer method.